src/app/geo-map/geo-map-routes-data.ts

```diff
@@ -0,0 +1,6 @@
+// tslint:disable:object-literal-sort-keys
+
+export const geomaproutesdata = {
```

Comment:
please always use camel case for variable names, e.g. GeoMapRoutesData

Reply:
will be fixed in my latest commit.